sql/item_mods.sql

```diff
@@ -36500,9 +36500,14 @@ INSERT INTO `item_mods` VALUES (18381,500,10); -- ITEM_ADDEFFECT_DMG: 10
 INSERT INTO `item_mods` VALUES (18381,501,5);  -- ITEM_ADDEFFECT_CHANCE: 5
 INSERT INTO `item_mods` VALUES (18381,950,1);  -- ITEM_ADDEFFECT_ELEMENT: 1
 
--- Algol
-INSERT INTO `item_mods` VALUES (18385,23,3);  -- ATT: 3
-INSERT INTO `item_mods` VALUES (18385,302,3); -- TRIPLE_ATTACK: 3
+-- Algol https://ffxiclopedia.fandom.com/wiki/Talk:Algol
+INSERT INTO `item_mods` VALUES (18385,23,3);    -- ATT: 3
+INSERT INTO `item_mods` VALUES (18385,302,3);   -- TRIPLE_ATTACK: 3
+INSERT INTO `item_mods` VALUES (18385,431,1);   -- ITEM_ADDEFFECT_TYPE: 1
+INSERT INTO `item_mods` VALUES (18385,499,1);   -- ITEM_SUBEFFECT: 1
+INSERT INTO `item_mods` VALUES (18385,500,20);  -- ITEM_ADDEFFECT_DMG: 20
+INSERT INTO `item_mods` VALUES (18385,501,46);  -- ITEM_ADDEFFECT_CHANCE: 46
```

Comment:
thats pretty high, mine never seemed to go off nearly half of its swings. got some data or source for that 46% rate?

Reply:
I went looking for data on the proc rate, and located one in the comments on 'clopedia. I added the source URL in the Algol comment just above this.

Reply:
I seriously do not trust comments ion shitclopedia. I'm going to take mine out later and waste a weekend swinging the hell out of it I guess...

I bet you it falls in line with the 3 theme, either a multiple of it or just 33%

since this is easy to change I'm not invested in what the number is when the pr merges, but def going to check it out further than some random's comment on the least reliable wiki ever.

Reply:
JP wiki ballparks it 40% ish (so we have "close enough" to call that 46) and the damage in the 30's, a little higher than you had.